docs/articles/commands/custom_context_checks.md

```diff
@@ -23,8 +23,11 @@ public class DirectMessageUsageAttribute : ContextCheckAttribute
 
 This is the attribute that you will apply to the command method.
```

Comment:
This feels a bit awkward. Perhaps instead:
*Above is the example attribute that we'll be using.*
Or maybe remove it altogether.